src/ConcreteOperations/intersection.jl

```diff
@@ -89,6 +89,40 @@ function intersection(L1::Line{N}, L2::Line{N}
     end
 end
 
+"""
+    intersection(X::LineSegment{N}, hp::Line{N}
+                ) where {N<:Real}
+Compute the intersection of a line and a line segment.
+### Input
+- `a`  -- LineSegment
+- `b` -- Line
+### Output
+If the sets do not intersect, the result is the empty set.
+Otherwise the result is the singleton or line segment that describes the intersection.
```

Comment:
Something happened to the empty lines (I have this problem when I copy code from Github).
The top line should also be a copy of the function signature (probably a copy-paste error here).